examples/design-system/apps/docs/stories/button.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import type { Meta, StoryObj } from "@storybook/react";
-import { Button } from "ui";
+import { Button } from "@acme/ui";
```

Comment:
Did we change the org in these names to `@repo` everywhere?

Reply:
Went with `@acme` on this one since it's got a publishing workflow in it.